decidim-core/lib/decidim/events/user_group_event.rb

```diff
@@ -35,9 +35,15 @@ def group_presenter
         end
 
         def group
-          return unless extra[:group].is_a?(Decidim::UserGroup)
+          @group ||= fetch_group
+        end
+
+        private
+
+        def fetch_group
+          return extra[:group] if extra[:group].is_a?(Decidim::UserGroup)
```

Comment:
I think we could remove this `extra[:group]` if we will not use it for nothing. What do you think @quinHD?